src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -19234,7 +19234,10 @@ public static void Main()
                 Diagnostic(ErrorCode.ERR_NameNotInContext, "_").WithArguments("_").WithLocation(14, 30),
                 // (18,58): error CS1615: Argument 1 may not be passed with the 'out' keyword
                 //             var list2 = new Dictionary<int, long> { [out _] = 6 };
-                Diagnostic(ErrorCode.ERR_BadArgExtraRef, "_").WithArguments("1", "out").WithLocation(18, 58)
+                Diagnostic(ErrorCode.ERR_BadArgExtraRef, "_").WithArguments("1", "out").WithLocation(18, 58),
+                // (9,18): error CS0165: Use of unassigned local variable 'x1'
```

Comment:
>// (9,18): error CS0165: Use of unassigned local variable 'x1' [](start = 16, length = 62)

Why did we start reporting this error?

Reply:
This was the result of reverting `PreciseAbstractFlowPass.VisitObjectInitializerMember` back to `master`. I've undone that change now.